dspace-jspui/src/main/webapp/home.jsp

```diff
@@ -134,7 +134,7 @@
 			<p>Easily share introductory statistics material&mdash;including <strong>copyright-cleared simulations, video, data sets</strong>, and more&mdash;with other educators and get meaningful feedback.</p>
 		    </div>
 		</div>
-	      
+	   <!--
```

Comment:
Is there a possibility that this section will be used in the future? If not, please remove. If yes, please put some comments for reference.